docs/src/tutorials/calibration.qmd

```diff
@@ -0,0 +1,94 @@
+
+On top of these issues, which stem directly from Bayes' theorem, with Bayesian neural network there is also the problems introduced by the approximate inference method adopted to compute the posterior distribution of the weights.\  To introduce the concept of Average Calibration and the Calibration Plots, we will follow closely the paper [Accurate Uncertainties for Deep Learning Using Calibrated Regression](https://arxiv.org/abs/1807.00263), written by Volodymyr Kuleshov, Nathan Fenner and Stefano Ermon, although with some small differences. We will hightlight these differences in the following paragraphs whenever they appear. We present here the theoretical basis necessary to understand the issue of calibration and we refer to the tutorials for the coding examples.
+
+## Notation
+We are given a labeled dataset $x_t, y_t \in X \times Y$ for $t = 1, 2, ..., T$ of i.i.d. realizations of random variables $X, Y \sim P$, where $P$ is the data distribution. 
+Given $x_t$, a forecaster $H : X \rightarrow (Y \rightarrow [0, 1])$ outputs at each step $t$ a CDF $F_t(y)$ targeting the label $y_t$. When $Y$ is continuous, $F_t$ is a cumulative probability distribution (CDF).  We will use $F^{−1}_t: [0, 1] → Y$ to denote the quantile function $F^{−1}_t (p) = inf\{y : p ≤ F_t(y)\}$.
+
+## Calibration in the Regression case
+In the regression case, we say that the forecaster H is  (on average) calibrated if
+
+$$\frac{\sum_{t=1}^T \mathbb{1} \{ y_t \leq F_t^{-1}(p)  \} }{T} \rightarrow p \quad \text{for all}\quad  p \in [0,1]$$
+
+as $T \rightarrow \infty$.\  In other words, the empirical and the predicted CDFs should match as the dataset size goes to infinity.
+Perfect Calibration is a sufficient condition for average calibration, the opposite however is not necessarily true: a model can be average calibrated but not perfectly calibrated.\
+From now on when we talk about calibration we will implicitly talk about average calibration rather than perfect calibration. 
+
+
+### Sharpness 
+Calibration by itself is not sufficient to produce a useful forecast. For example, it is easy to see that if we use for the forecast the marginal distribution $F(y) = \mathbb{P}(Y ≤ y)$, without considering the input feature $X$, the forecast will be calibrated but still not accurate. In order to be useful, forecasts must also be sharp, which,in a regression context, means that the confidence intervals should all be as tight as possible around a single value. More formally, we want the variance $var(F_t)$ of the random variable whose CDF is $F_t$ to be small.\
```

Comment:
"if we use the marginal distribution $F(y) = \mathbb{P}(Y ≤ y)$ for the forecast without ..."

Reply:
"which,in a regression context, means" -> "which (in a regression context) means ..."